src/lib/tabs/tab-group.spec.ts

```diff
@@ -279,7 +279,7 @@ describe('MatTabGroup', () => {
         tick();
 
         tabs = component._tabs.toArray();
-        expect(tabs[3].origin).toBeGreaterThanOrEqual(0);
+        expect(tabs[3].origin! >= 0).toBe(true);
```

Comment:
Can you explain what's going on here?

Reply:
This was left over from when I was reviewing #10642, will remove